sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,2 +1,350 @@
+project exists in. Last, the related GitHub repository will remain intact after
+deleting a project from Semaphore 2.0.
+
+Last, you can use the same project name under multiple organizations but you
+cannot use the same project name more than once under a single organization.
+
+## Flags
+
+### The --help flag
+
+The `--help` flag, which can also be used as `-h`, is used for getting
+information about a `sem` command or `sem`.
+
+### The --verbose flag
+
+The `--verbose` flag, which can also be used as `-v` displays verbose
+output – you will see the interaction and the data exchanged between
+`sem` and the Semaphore 2.0 servers.
+
+The `-v` flag is useful for debugging.
+
+### The -f flag
+
+The `-f` flag allows you to specify the path to the desired YAML file that will
+be used with the `sem create` command in order to create a new `secrets` bucket
+or a new Semaphore 2.0 project.
+
+## Examples
+
+This section will present examples for all `sem` commands.
+
+### sem version
+
+The `sem version` command displays the used version of the `sem` tool. As an
+example, if you are using `sem` version 0.4.1, the output of `sem version`
+will be as follows:
+
+    $ sem version
+    v0.4.1
+
+Your actual output might be different on your machine.
+
+Additionally, the `sem version` command does not work with the `-f` flag and
+does not create any additional output when used with the `-v` flag.
+
+### sem help
+
+The output of the `sem help` command is static and identical to the output of
+the `sem` command when executed without any command line arguments.
+
+Additionally, the `help` command can be also used as follows (the `connect`
+command is used as an example here):
+
+    sem connect help
+
+In that case, `help` will generate information about the use of the
+`sem connect` command.
+
+### sem connect
+
+The `sem connect` command should be executed as follows:
+
+    sem connect tsoukalos.semaphoreci.com NeUFkim46BCdpqCAyWXN
+
+### sem context
+
+The next command will just list all the organizations the connected user
+belongs to:
+
+    sem context
+
+In the output of `sem context`, the active organization will have a `*`
+character in front of it.
+
+If you use an argument with `sem context`, then that argument should be a valid
+organization name as it appears in the output of `sem context`. So, in order to
+change from the current organization to `tsoukalos_semaphoreci_com`, you should
+execute the next command:
+
+    sem context tsoukalos_semaphoreci_com
+
+### sem create
+
+The `sem create` command goes hand by hand with the `-f` flag because you
+always need to provide a valid YAML file with the `sem create` command. So,
+if you have a valid secret or project YAML file stored at `/tmp/valid.yaml`,
+you should execute the next command in order to add that `secrets` bucket or
+project under the current organization:
+
+    sem create -f /tmp/valid.yaml
+
+### sem delete [project | projects]
```

Comment:
@shiroyasha do we want this ability to specify both singular and plural for `sem delete`? Reading it in the documentation kind of leaves me wondering why do we need this?

Is this also possible?

> sem delete projects projects1 project2 projectN

Overall I don't have problem with it just wondering if we should document it? I am not sure if it's helping users.

Reply:
Also in examples below having both singular and plural examples feels just like a noise. What do you think @mactsouk @shiroyasha ?

Reply:
It is a noise but I think that it should be documented.

Reply:
```
sem delete projects newRepo secretsInvalid
Error: accepts 2 arg(s), received 3
Usage:
$ sem delete [KIND] [NAME] [flags]

Flags:
  -h, --help   help for delete

Global Flags:
  -v, --verbose   verbose output

accepts 2 arg(s), received 3
```

So, it does not work.

Reply:
> The official name of the kinds are `secret` and `project`.

So I am going to go with the official names. Thanks @shiroyasha.

Reply:
Done.